File: online_judge/assets/compile1.java
import java.util.Scanner;

public class CalculateSum {
    public static void main(String[] args) {
        double num1, num2;
        Scanner scanner = new Scanner(System.in);

        // Input the first number
        num1 = scanner.nextDouble();

        // Input the second number
        num2 = scanner.nextDouble();

        // Close the scanner to release resources
        scanner.close();

        // Add two numbers
        double sum = num1 + num2;

        // Display the sum
        System.out.println(sum);
    }
}

File: online_judge/assets/compile1.cpp
#include <iostream>

int main() {
    try {
        // Input the first number
        double num1;
        std::cin >> num1;

        // Input the second number
        double num2;
        std::cin >> num2;

        // Add two numbers
        double sum_result = num1 + num2;

        // Display the sum
        std::cout << sum_result << std::endl;
    }
    catch (const std::exception& e) {
        std::cout << "Invalid input. Please enter valid numbers." << std::endl;
    }

    return 0;
}
